api/v1alpha1/nnfstorageprofile_types.go

```diff
@@ -137,49 +140,47 @@ type NnfStorageProfileCmdLines struct {
 	// LvCreate specifies the lvcreate commandline, minus the "lvcreate".
 	LvCreate string `json:"lvCreate,omitempty"`
 
+	// LvChange specifies the various lvchange commandlines, minus the "lvchange"
+	LvChange NnfStorageProfileLVMLvChangeCmdLines `json:"lvChange,omitempty"`
+
 	// LvRemove specifies the lvcreate commandline, minus the "lvremove".
 	LvRemove string `json:"lvRemove,omitempty"`
+
+	// MountRabbit specifies mount options for mounting on the Rabbit.
+	MountRabbit string `json:"mountRabbit,omitempty"`
```

Comment:
Are we using MountRabbit and MountCompute anymore?

Reply:
I should add it. It wasn't plumbed in to Clientmountd before, but it should be pretty straightforward to get it working now.

Reply:
Right, we never did use MountCompute (oops).  But we were using MountRabbit, and that's been lost.